deps/rabbitmq_mqtt/test/reader_SUITE.erl

```diff
@@ -261,6 +261,8 @@ event_authentication_failure(Config) ->
 %% Test that queue type rabbit_mqtt_qos0_queue drops QoS 0 messages when its
 %% max length is reached.
 rabbit_mqtt_qos0_queue_overflow(Config) ->
+    ok = rabbit_ct_broker_helpers:enable_feature_flag(Config, rabbit_mqtt_qos0_queue),
```

Comment:
This will crash if the feature flag can't be enabled.

You should enable the feature flag(s) you need from an `init_per_*()` function and decide what do next.

If the feature flag can't be enabled, then either skip the testcase if it doesn't make sense without it, or do something different if the main test scenario can't work without it.

Reply:
Ah ok, I missed the commit message, sorry.